easybuild/tools/module_generator.py

```diff
@@ -693,6 +711,29 @@ def getenv_cmd(self, envvar):
         return 'os.getenv("%s")' % envvar
 
 
+    def set_as_default(self, module_folder_path, module_version):
+        """
+        Create file called "default" inside the package's module folder in order
+        to set the default module version
+        :param module_folder_path: module folder path, e.g. $HOME/easybuild/modules/all/Bison
+        :param module_version: module version, e.g. 3.0.4
+        """
+        default_filepath = os.path.join(module_folder_path, 'default')
+
+        if os.path.islink(default_filepath):
+            points_to = readlink(default_filepath)
+            remove_file(default_filepath)
+            self.log.info("Removed default version marking from %s.", points_to)
+        elif os.path.exists(default_filepath):
+            raise EasyBuildError('Found an unexpected file called default in dir %s' % module_folder_path)
+
+        cwd = os.getcwd()
+        #os.chdir(module_folder_path)
+        symlink(module_version + self.MODULE_FILE_EXTENSION, default_filepath)
+        #os.chdir(cwd)
```

Comment:
please clean up the uncommented code, and you can drop the `cwd = ` line too (not used anymore)